tools/check_format.py

```diff
@@ -11,7 +11,7 @@
                      "./bazel-")
 SUFFIXES = (".cc", ".h", "BUILD")
 
-CLANG_FORMAT_PATH = os.getenv("CLANG-FORMAT", "clang-format-3.6")
+CLANG_FORMAT_PATH = os.getenv("CLANG_FORMAT", "clang-format-3.6")
```

Comment:
Sure, but could you elaborate. I needed this change to be able to run the format check, which was failing, and passing it seemed to be a prerequisite for the PR to be mergeable. I understand that in ci 'clang-format-3.6' must exist.